pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html

```diff
@@ -403,7 +403,7 @@ <h5 i18n="ldap user search attributes title" aria-level="3">
                             <clr-tooltip>
                                 <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22"></clr-icon>
                                 <clr-tooltip-content clrPosition="top-left" clrSize="lg" *clrIfOpen>
-                                    <span>The LDAP attribute that contains the username used in Kubernetes. In AD environments
+                                    <span>The LDAP attribute that contains the username used in Kubernetes and searched by CN only for now. In AD environments
```

Comment:
Please change to:

The LDAP attribute that contains the username used in Kubernetes; can only be searched by CN.

Reply:
fixed